pkg/rtc/mediatrackreceiver.go

```diff
@@ -240,7 +240,7 @@ func (t *MediaTrackReceiver) SetLayerSsrc(mime string, rid string, ssrc uint32)
 	defer t.lock.Unlock()
 
 	layer := buffer.RidToSpatialLayer(rid, t.params.TrackInfo)
-	if layer == sfu.InvalidLayerSpatial {
+	if layer == buffer.InvalidLayerSpatial {
```

Comment:
Initially, these were declared in `sfu/forwarder.go`. Then they were moved to `buffer/videolayer.go`. But, there was a type and some variable proxying in `sfu` namespace. Just removing the proxy and using the one true definition of these.